mypy/build.py

```diff
@@ -1016,12 +1016,15 @@ def read_deps_cache(manager: BuildManager,
 def _load_json_file(file: str, manager: BuildManager,
                     log_sucess: str, log_error: str) -> Optional[Dict[str, Any]]:
     """A simple helper to read a JSON file with logging."""
+    t0 = time.time()
     try:
         data = manager.metastore.read(file)
     except IOError:
         manager.log(log_error + file)
         return None
-    manager.trace(log_sucess + data.rstrip())
+    manager.add_stats(metastore_read_time=time.time() - t0)
+    if manager.verbosity() >= 2:
```

Comment:
This `if` is not needed, `trace()` already has it.

Reply:
The idea is to avoid stripping and concatenating the strings

Reply:
Oh, I see! I would then add a comment about this.